### ISSUE TITLE
FR: Bookmark ribbon instead of dog ear

### ISSUE THREAD

**Issue by slater1** [koreader/koreader#9965]
TLDR: Option to use a ribbon instead of dog-ear to indicate bookmarks

My wife is a librarian, and she noticed while I was reading a book in koreader that the bookmark icon in the top right corner of a bookmarked page imitates a dog-eared page. She was dismayed because for real books, this is a frowed-upon practice that damages books and costs libraries real money. 

I showed her that the icon to access the bookmarks menu uses a book ribbon icon, and we wondered why the same icon could not be shown when viewing a page that is bookmarked. In our opinion, the current iconography normalizes / perpetuates the abuse of books.

BTW, thanks for a great app that I use every day.

**Comment by hius07** [comment]
Thank you, interesting point!

**Comment by Frenzie** [comment]
It might look a bit like a dogear but it's actually this concept. Maybe. ;-p

![image](https://user-images.githubusercontent.com/202757/209873378-cfb99007-db89-44aa-b4d2-6d59fc4170de.png)

**Comment by poire-z** [comment]
> In our opinion, the current iconography normalizes / perpetuates the abuse of books.

In my opinion, from when I used to buy used books, underlining text and drawing lines in the margins is a far bigger crime than dog-earing page corners !
So, I suggest we remove all text highlighting/underlining from KOReader :)
And that we go start `#ittoo` !

**Comment by Frenzie** [comment]
I once borrowed a book from the library where someone had underlined with an exclamation mark all words they thought were bad.

Another time I borrowed a French book and someone had written incorrect translations in the margin.

I agree that the occasional presumably accidental dogear is barely noticable in comparison.

**Comment by slater1** [comment]
Thanks poire-z, for helping me see my post in the proper perspective.

Clearly, the rules for real books are different than e-books. Underlining, highlighting-and dog-earing an EPUB does not modify the EPUB so it doesn't reduce anyone else'e experience, even if it was used through an e-library.

Even so, does [skeumorphism](https://en.wikipedia.org/wiki/Skeuomorph) influence peoples' attitudes toward real objects? Perhaps this is like arguing whether to call the place I delete files the Trash Can or Recycle Bin. Ecologically it doesn't matter, but it communicates a view about the destiny of discarded data/files.

Feel free to close, I don't need to advocate for this feature.

**Comment by roygbyte** [comment]
Reading this PR this morning has reminded me that dogeared pages, when used responsibly, can be oh-so-satisfying to make.

**Comment by Frenzie** [comment]
I only use dogears (although I think of them as donkey ears) in ads, from the regular weekly nonsense to mail order catalogs.

**Comment by Commodore64user** [comment]
I remember @poire-z telling me once how to properly patch the doggy ear file, but I can't find it anymore... care to remind my forgetful memory? please.

**Comment by poire-z** [comment]
(My memory and search skills are as bad as yours...)

**Comment by Commodore64user** [comment]
Couldn't we, not re-download all those images if they are already installed? That would allow for users to switch one icon for another and to survive firmware updates and duplicates of unused icons.

**Comment by Frenzie** [comment]
The update mechanism uses zsync2, so generally it doesn't. However, it'll still overwrite them.

**Comment by Commodore64user** [comment]
I can confirm, that there is a brighter future... that dog's-ears don't have to be tiny and ugly simply because @poire-z thinks that is good enough, we shall one day be free, in the meantime, one can create a folder called icons in the `koreader/icons` directory and drop just about anything you want with the name `dogear_alpha`, koreader will use that image instead

<p align="center">
<img src="https://github.com/user-attachments/assets/ab0ae199-8005-49a9-8603-25b9398f1c0d" width=40%> 
</p> 

as for making it bigger, like for real men (or women, no discrimination) ;), I can share the patch I have later, should you wish to have it
